src/fiatconnect/saga.ts

```diff
@@ -969,7 +972,16 @@ export function* _initiateSendTxToProvider({
       transferAddress: transferAddress,
       provider: fiatConnectQuote.getProviderId(),
     })
-    throw error
+    // If we've timed out, or the error is deemed unsafe to retry,
+    // it's possible that the transaction is already processing. Note that
+    // this check is not perfect; there may be false positives/negatives.
+    const txPossiblyPending =
+      error?.message === ErrorMessages.TRANSACTION_TIMEOUT || isTxPossiblyPending(error)
```

Comment:
seems like this message check should also be done in the isTxPossiblyPending helper